assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -278,7 +278,7 @@ export const createSettingsStore = ( type, identifier, datapoint, {
 		 * @return {Object} Redux-style action.
 		 */
 		actions[ `set${ pascalCaseSlug }` ] = ( value ) => {
-			invariant( typeof value !== 'undefined', 'value is required.' );
+			invariant( typeof value !== 'undefined', `value is required for calls to set${ pascalCaseSlug }().` );
```

Comment:
I added this to assist in debugging; because these method names are dynamically generated, this makes it a lot easier to discover which method is throwing the error, which at first made narrowing down the `invariant` call cumbersome.